.travis.yml

```diff
@@ -1,10 +1,6 @@
 udo: false
 language: node_js
 node_js:
-  - 4
-  - 5
-  - 6
-  - 7
```

Comment:
The tests aren't put through babel so things like `const` made it fall over. Adding babel just for this feels like overkill. The tests are meant to check if our specific rules behave as intended not if the rules behave the same across all versions of Node (which I think is outwith the scope of a config).